docusaurus.config.js

```diff
@@ -67,8 +68,36 @@ const config = {
 
   plugins: [
     './plugins/fav-icon',
+    async function myPlugin(context, options) {
```

Comment:
Is it ok to rename this function to something more specific?

Reply:
yep!